include/psp2/ctrl.h

```diff
@@ -28,13 +28,17 @@ enum {
 /** Enumeration for the digital controller buttons. */
 enum {
 	SCE_CTRL_SELECT     = 0x000001,	//!< Select button.
+	SCE_CTRL_L3         = 0x000002,	//!< L3 button.
+	SCE_CTRL_R3         = 0x000004,	//!< R3 button.
 	SCE_CTRL_START      = 0x000008,	//!< Start button.
 	SCE_CTRL_UP         = 0x000010,	//!< Up D-Pad button.
 	SCE_CTRL_RIGHT      = 0x000020,	//!< Right D-Pad button.
 	SCE_CTRL_DOWN       = 0x000040,	//!< Down D-Pad button.
 	SCE_CTRL_LEFT       = 0x000080,	//!< Left D-Pad button.
 	SCE_CTRL_LTRIGGER   = 0x000100,	//!< Left trigger.
 	SCE_CTRL_RTRIGGER   = 0x000200,	//!< Right trigger.
+	SCE_CTRL_L1         = 0x000400,	//!< L1 button.
```

Comment:
in sceCtrlReadBufferPositiveExt2, L2 button set 0x100(LTRIGGER) but L1 button set 0x400.
(also R2 set 0x200, R1 set 0x800)
but general used sceCtrlPeekBufferPositive, only can use LT/RT values.

can you suggest better name if it looks confused?